client/views/room/contextualBar/PruneMessages/PruneMessages.js

```diff
@@ -218,8 +218,12 @@ export default ({
 			if (result < 1) {
 				throw new Error(t('No_messages_found_to_prune'));
 			}
+			if (result === 1) {
+				dispatchToastMessage({ type: 'success', message: `${ result } ${ t('message_pruned') }` });
+			} else {
+				dispatchToastMessage({ type: 'success', message: `${ result } ${ t('messages_pruned') }` });
+			}
```

Comment:
You can change this to:
```
const message = result === 1 ? 'message_pruned' : 'messages_pruned';
dispatchToastMessage({ type: 'success', message: `${ result } ${ t(message) }` });
```

Reply:
@KevLehman I have made the requested changes. But as the repo linked to the pull request was deleted I have made another pull request #21902